doc/rest.md

```diff
@@ -287,7 +291,7 @@ JSON object, same as in case of [GET Workspace Layer](#get-workspace-layer), pos
 - *files_to_upload*: List of objects. It's present only if **file** parameter contained file names. See [POST Workspace Layers](#post-workspace-layers) response to find out more.
 
 ### DELETE Workspace Layer
-Delete existing layer and all associated sources, including vector data file and DB table. It is possible to delete layer, whose publication process is still running. In such case, the publication process is aborted safely.
+Delete existing layer and all associated sources, including vector data file and DB table. [Asynchronous tasks](async-tasks.md) of affected layer are aborted.
```

Comment:
'Asynchronous tasks of affected layer are aborted.' -> 'Currently running asynchronous tasks of affected layer are aborted.'